src/common/components/modal/styles.module.scss

```diff
@@ -0,0 +1,85 @@
+.modal {
+	align-items: center;
+	background-color: rgba($color: #333, $alpha: 0.6);
```

Comment:
maybe you should to add this color to the variables

Reply:
I tried, but it doesn't work. Maybe you'll be able to fix it

Reply:
i will try, if i don't forget